app/uk/gov/hmrc/hmrcemailrenderer/templates/itr/itrEmailVerification.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Verify your email address") {
+<p style="margin: 0 0 30px; font-size: 19px;">You’re getting this email because you’re applying to use a venture capital scheme.
+    We need you to confirm your email address so you can use the service.</p>
+<p style="margin: 0 0 30px; font-size: 19px;">To verify your email address click the link below:</p>
+<p style="margin: 0 0 30px; font-size: 19px;"><a href="@{params("verificationLink")}" style="color: #005EA5;">@{params("verificationLink")}</a></p>
+<p style="margin: 0 0 30px; font-size: 19px;">From HMRC</p>
```

Comment:
Please do not use 'From HMRC'.
See our content recommendations:
https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations